### ISSUE TITLE
Having to use product.content.id

### ISSUE THREAD

**Issue by picklenick144** [jcberquist/stripe-cfml#52]
Hi, as someone who's both relatively new to coldfusion and stripe, thanks for this repo! I wanted to make sure this is the expected functionality because the readme says:

stripe-cfml is modeled after the official Stripe SDKs. In particular it copies the class and method names used in the Node SDK. The Node examples given in the official Stripe documentation are simply able to be copied and used (with the notable difference that this library does not make use of callbacks - everything is done synchronously).

However I was trying to implement this dummy example: https://docs.stripe.com/development/quickstart?lang=node

```
stripe.products.create({
  name: 'Starter Subscription',
  description: '$12/Month subscription',
}).then(product => {
  stripe.prices.create({
    unit_amount: 1200,
    currency: 'usd',
    recurring: {
      interval: 'month',
    },
    product: product.id,
  }).then(price => {
    console.log('Success! Here is your starter subscription product id: ' + product.id);
    console.log('Success! Here is your starter subscription price id: ' + price.id);
  });
});
```

I modified it to:
```
    product = stripe.products.create({
        name: 'Starter Subscription',
        description: '$12/Month subscription'
    });

      
           
                price = stripe.prices.create({
                    unit_amount: 1200,
                    currency: 'usd',
                    recurring: { interval: 'month' },
                    product: product.content.id
                });
        
                    writeOutput("Success! Here is your starter subscription product id: " & product.content.id & "<br>");
```

The thing that really tripped me up was having to use `product.content.id` instead of `product.id` - I wanted to confirm this was expected?

**Comment by jcberquist** [comment]
You are right that the product data is actually contained under the content key in the response, see here: https://github.com/jcberquist/stripe-cfml?tab=readme-ov-file#responses - I wanted to return the response headers as well as the response content, so I opted for this format.

I can see that is somewhat confusing given the text you quoted claims that the Node examples are able to be simply copied and used. I could clarify that while the method signatures match when calling the API ("it copies the class and method names used in the Node SDK"), the response does not match the response you get from the Node SDK?

**Comment by picklenick144** [comment]
Yeah I think that would be really helpful for future users - thanks again! I'm working on integrating stripe into our coldfusion-based site and this has been really helpful.
